lib/cursor.js

```diff
@@ -136,3 +136,13 @@ module.exports = class Cursor extends Readable {
       });
   }
 }
+
+// Feature-detect async iterator protocol support, only available in Node 10+.
+if (typeof Symbol.asyncIterator === 'symbol') {
```

Comment:
The `else` case causes a reduction in coverage; I could add an `istanbul ignore else` comment or I could try and finagle the mocha/babel integration to allow the test to run without the polyfill, but that would be a substantial increase in complexity for just a tiny change in coverage.